keras/backend/tensorflow_backend.py

```diff
@@ -113,6 +113,32 @@ def _to_tensor(x, dtype):
     return x
 
 
+def is_sparse(tensor):
+    return isinstance(tensor, tf.SparseTensor)
+
+
+def how_sparse(tensors):
+    any_sparse = False
+    all_sparse = True
+    for tensor in tensors:
+        if is_sparse(tensor):
+            any_sparse = True
+        else:
+            all_sparse = False
+
+    return (any_sparse, all_sparse and any_sparse)
+
+
+def to_dense(tensors):
```

Comment:
Sure this should take a single tensor as input for API consistency. Using `[K.to_dense(x) for x in tensors]` is also not a big overhead for the batch version.